src/eureka/S3_data_reduction/s3_reduce.py

```diff
@@ -233,8 +264,8 @@ def reduce(eventlabel, ecf_path=None, s2_meta=None):
 
                 # Compute 1D wavelength solution
                 if 'wave_2d' in data:
-                    data['wave_1d'] = (['x'],
-                                       data.wave_2d[meta.src_ypos].values)
+                    data['wave_1d'] = (['x'], np.median(
```

Comment:
So this difference was driven by the default behaviour of `xarray.concat` which uses `coords='different'` and `compat='equals'` where data arrays that are not equal end up being concatenated along a new axis rather than `astraeus.xarrayIO.concat`'s default behaviour which uses `coords='minimal'` and `compat='override'` where arrays that are not equal end up being overridden by the first dataset. So with my `xarray.concat` call, `wave_2d` became time dependent when running the code in batch mode, so I'm taking the time average (along the time-axis, 0) and still using the `meta.src_ypos` index as was done before. If I switch to `xrio.concat` and use its default behaviour as you suggested above, then I won't need to do this (for better or for worse). Let me know which you prefer

Reply:
For now I just went with the `xrio.concat` option, but let me know if you want me to go back to the other behaviour

Reply:
I'm stuck on why we would want to compute a time-dependent wavelength solution. Maybe we should talk, if you think there's an advantage.

Reply:
I guess it doesn't really make sense does it - I'll stick with your suggested xrio.concat option